src/main/java/digital/slovensko/autogram/server/SignEndpoint.java

```diff
@@ -25,6 +25,7 @@ public void handle(HttpExchange exchange) throws IOException {
         try {
             var body = EndpointUtils.loadFromJsonExchange(exchange, SignRequestBody.class);
             body.validateDocument();
+            body.resolveSigningLevel();
```

Comment:
nedava zmysel tuto metodu strcit interne do validateSigningParameters? pride mi to taky temporal coupling, ze ked to nezavolas tak sa to nebude spravat dobre.

Reply:
na druhej strane, od validate by som čakal, že nebude meniť ten objekt. Ale je pravda, že je to idempotentné, takže by to nemusel byť problém.

Reply:
Pravda, najvacsi zmysel by to davalo uplne hore niekde kde sa to body/parametre vyrabaju.

Reply:
To áno, ale tieto classy sa vyrábajú trochu magicky cez ten gson parser a nechcel som ísť teraz do toho.

Reply:
hej chapem, strcme to do toho validate zatial s nejakou poznamkou